### ISSUE TITLE
Add optimizers to the auto_dl

### ISSUE THREAD

**Issue by vfp1** [vfp1/UPC-AIDL-bAbI#2]
Add Keras Optimizers to the auto_dl to control learning rate: https://keras.io/optimizers/
